api/src/pcapi/core/offers/api.py

```diff
@@ -1925,6 +1960,78 @@ def move_event_offer(
         on_commit(partial(transactional_mails.send_email_for_each_ongoing_booking, offer))
 
 
+def move_event_collective_offer(
+    offer: educational_models.CollectiveOffer, destination_venue: offerers_models.Venue
```

Comment:
```suggestion
    collective_offer: educational_models.CollectiveOffer, destination_venue: offerers_models.Venue
```
C'est plus pratique pour chercher dans le code après :)